e2e/src/rabbitmq_e2e.rs

```diff
@@ -625,3 +645,57 @@ pub async fn rabbitmq_e2e_test() -> Result<(), Error> {
     println!("E2e test passed.");
```

Comment:
You can probably remove scaling_test() from this one since you already have a rabbitmq_scaling_e2e_test() below

Reply:
sorry just forgot